include/swift/Basic/Version.h

```diff
@@ -153,7 +153,7 @@ class Version {
   // List of backward-compatibility versions that we permit passing as
   // -swift-version <vers>
   static std::array<StringRef, 4> getValidEffectiveVersions() {
```

Comment:
Array size should be 3 instead of 4
```suggestion
   static std::array<StringRef, 3> getValidEffectiveVersions() {
```